modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -594,6 +594,22 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="angle_diff">
+			<return type="float">
+			</return>
+			<argument index="0" name="minuend" type="float">
+			</argument>
+			<argument index="1" name="subtrahend" type="float">
+			</argument>
+			<description>
+				Returns the angle difference (within the range of -PI to PI) between minuend and subtrahend. All angles
+				are in radian.
+				[codeblock]
+				angle_diff(PI*1.5,0) # Returns -1.570796 same as -PI*0.5
+				rad2deg(angle_diff(deg2rad(270),deg2rad(0))) # Returns -90
```

Comment:
Spaces after the `,` and between the `*`s would be nice.

Reply:
done